### DISCUSSION TITLE
2.3 Release schedule

### DISCUSSION THREAD

**Discussion by joelhess** [awspring/spring-cloud-aws#85] [General]
Is there any update on the scheduled release for 2.3? I see a couple new issues attached the milestone related to FIFO support. Would it be possible to get a 2.3.0 with the spring boot 2.4 compatibility and a fast follow on with those fixes?

**Comment by maciejwalkowiak** [comment]
2.3 GA will be released in upcoming days. We got busy with other duties 🤷‍♂️

**Reply by maciejwalkowiak** [reply]
2.3 got released https://github.com/awspring/spring-cloud-aws/releases/tag/v2.3.0
